Sources_v3/StreamChatUI/ChatChannel/MessageComposer/MessageComposerSuggestionsViewController.swift

```diff
@@ -97,9 +74,9 @@ open class MessageComposerSuggestionsViewController<ExtraData: ExtraDataTypes>:
             options: [.new],
             changeHandler: { [weak self] _, change in
                 DispatchQueue.main.async {
-                    guard let newSize = change.newValue, newSize.height < 300 else {
+                    guard let newSize = change.newValue, newSize.height < 230 else {
                         // TODO: Compute size better according to 4 cells.
-                        self?.view.frame.size.height = 300
+                        self?.view.frame.size.height = 230
```

Comment:
nit: The `4` should be a constant which should be modifiable via default appearance hook (like we do it for intrinsic content size elsewhere)